### PR TITLE
docs: Provide information about Studio Public IPs

### PR DESCRIPTION
**Pull request by mjasion**

### CODE REVIEW

File: content/docs/studio/user-guide/connect-custom-gitlab-server.md [hunk 1]
```diff
@@ -49,3 +49,20 @@ Custom GitLab server support is available only within a team workspace, and is
 available to all Iterative Studio teams (free and paid).
 
 </admon>
+
+## My GitLab server is behind a firewall. What should I do?
```
Comment (shcheklein) [inline content/docs/studio/user-guide/connect-custom-gitlab-server.md:53]: Can be also useful / was asked for for for the on-prem storage (SSH).

**Review by shcheklein — COMMENTED** [review]
Thanks @mjasion ! One minor addition, otherwise looks good to me.

File: content/docs/studio/user-guide/connect-custom-gitlab-server.md (hunk 1 above)
Comment (mjasion) [reply content/docs/studio/user-guide/connect-custom-gitlab-server.md:53]: Oh, I didn't know that. Let me add this somewhere ;-)

**Review by shcheklein — APPROVED** [review]

File: content/docs/studio/user-guide/connect-custom-gitlab-server.md (hunk 1 above)
Comment (mjasion) [reply content/docs/studio/user-guide/connect-custom-gitlab-server.md:53]: @shcheklein can you re-check again?

**Comment by 0x2b3bfa0** [comment]
Closes https://github.com/iterative/itops/issues/1190

**Pull request merged**